filters/H5Zbitshuffle/src/H5Zbitshuffle.jl

```diff
@@ -0,0 +1,280 @@
+#==
```

Comment:
Make this a docstring so that `?H5Zbitshuffle` works.

Reply:
Did this in the latest commit.